examples/sellCoin.ts

```diff
@@ -0,0 +1,17 @@
+import { createWalletClient, http, parseEther } from "viem";
+import { ParagraphAPI } from "../src";
+import { base } from "viem/chains";
+import { privateKeyToAccount } from "viem/accounts";
+
+const api = new ParagraphAPI();
+const client = createWalletClient({ chain: base, transport: http() });
+const account = privateKeyToAccount("0x.."); // Add your private key for this example to work
```

Comment:
Can you add one more example for buying & selling, but _not_ using the `privateKeyToAccount`?

I imagine most implementors would not be providing their own private key or account, but would require their own users to sign a txn to actually buy.